src/picture_widget.rs

```diff
@@ -23,6 +23,7 @@ use gelatin::{DrawContext, Event, EventKind, Widget, WidgetData, WidgetError};
 
 use std::time::{Duration, Instant};
 
+static TOGGLE_FULLSCREEN: &'static str = "toggle_fullscreen";
```

Comment:
`TOGGLE_FULLSCREEN` should be called `TOGGLE_FULLSCREEN_NAME` to be consistent with the rest.

Reply:
Fixed